research/src/pages/select/select.proposal.mdx

```diff
@@ -117,6 +117,30 @@ children:
+
+The different parts of the `<select>` element and its popup `<listbox>` must expose the following default accessibility roles, states and properties: 
+
+
+| Part       | Role                   | State/Property Attribute     | Usage                                                                               |
+| -----------| -----------------------| -----------------------------|------------------------------------------------------------------------------------ |
+| select     |                        |                              | Serves as a wrapper to its accessible parts                                         |
+| button     | `combobox`             |                              | Exposes the necessary `combobox` role                                                |
+|            |                        | `aria-controls=IDREF`        | Creates association with the `listbox`                                              |
+|            |                        | `aria-expanded=true / false` | Identifies if the `listbox` popup is displayed. Reflects the `open` attribute       |
+|            |                        | `aria-haspopup=listbox`      | Identifies that the control will invoke a listbox popup. *Note* the [value of this attribute may change](https://github.com/w3c/aria/issues/1024#issuecomment-1011303969).      |
+|            |                        | `aria-controls=IDREF`        | Creates association with the `listbox`                                              |
+| listbox    | `listbox`              |                              | Exposes the necessary `listbox` role                                              |
+| option     | `option`               |                              | Exposes the necessary `option` role                                             |
+| optgroup   | `group`                |                              | xposes the necessary `group` role                                              |
+  
+**Note / open question:** the `<button>` part will need to use `aria-activedescendant` if keyboard focus is to remain on that element when the `<listbox>` popup is invoked. If keyboard focus is actually set to one of the `<option>` elements within the `<listbox>`, then this may not be necessary and focus management can be handled differently.
+
+**Question:** should all accessibile states and properties be covered here?  e.g., `disabled`, `selected`, etc.?  If `<select>` will retain multiselect functionality, it seems reasonable. If not, then it may be beyond what's necessary.
```

Comment:
I think that we'll eventually want this to be comprehensive, it's reasonable to cover `disabled` and `selected` behavior for single-select. Doesn't need to be part of this change though.

Reply:
I don't mind adding it.  I just wanted to make sure it was desired before I did more than what was needed.  thanks

Reply:
added in content for `disabled`, `selected` and `required`.   Added in expected behaviors for when selecting an option / dismissing a popup listbox.

Question still remains on the way focus will be treated, but the outlined behaviors would apply in either case (they might just be less relevant if the invoking combobox/button retains focus and `aria-activedescendant` is used to indicate the current selected option).

Expectations for `multiple` are left out for now until resolution of #447